main.scss

```diff
@@ -167,24 +167,24 @@
 
 			// this adapts to fit the screen based on left o-grid-row margins
 			&:after {
-				margin-left: calc(-50vw + 5px);
 				@include oGridRespondTo(M) {
 					margin-left: calc(-50vw + 10px);
-				};
+				}
+				margin-left: calc(-50vw + 5px);
 			}
 
 			// modifies borders when stacked(S) and horizontal(M+)
 			.o-teaser-collection__item {
-				margin: 0;
-				border-bottom: 1px solid rgba(oColorsByName('white'), 0.2);
-				padding: 15px 0;
 
 				@include oGridRespondTo(M) {
```

Comment:
The `oGridRespondTo` mixin outputs a media query and so should come after other declarations, unlike other mixins. Media queries add no specificity to the selectors they contain, so source order matters.

I _think_ this is ok because:
```
.a{
  @media (min-width: 600px) {
    background: red;
  }
  background: blue;
}
```
becomes
```
.a {
  background: blue;
}
@media (min-width: 600px) {
  .a {
    background: red;
  }
}
```

But having the media query after the declarations I find more readable, so I'd rather we play it safe and revert this. Was it required? Or `--fix` being unreasonably helpful?

Reply:
it was all done by `--fix`

Reply:
This was my mistake 😞 obt has a flawed test
https://github.com/Financial-Times/stylelint-config-origami-component/pull/2